validator/accounts/v2/wallet_create.go

```diff
@@ -32,23 +32,23 @@ func CreateWallet(cliCtx *cli.Context) (*Wallet, error) {
 		if err = createDirectKeymanagerWallet(cliCtx, w); err != nil {
 			return nil, errors.Wrap(err, "could not initialize wallet with direct keymanager")
 		}
-		log.WithField("wallet-path", w.walletDir).Info(
+		log.WithField("wallet-dir", w.walletDir).Info(
```

Comment:
I think you should include `--` at the beginning, otherwise copy-pasting the command will not recognize `wallet-dir` as a flag.